integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,271 @@
+
+For example, creating packages for `arch=armv8`:
+
+The profile will be:
+
+.. code-block:: text
+   :caption: *armv8*
+
+    [settings]
+    os_build=Linux
+    arch_build=x86_64
+    os=Linux
+    arch=armv8
+    compiler=gcc
+    compiler.version=8
+    compiler.libcxx=libstdc++11
+    build_type=Release
+
+Activate the SDK environment and execute the create command.
+
+.. code-block:: bash
+
+    $ source oe-environment-setup-aarch64-poky-linux
+    $ conan create . user/channel --profile armv8
+
+This will generate the packages using the Yocto toolchain. Now you can upload the binaries to an Artifactory server so share and reuse in
+your Yocto builds.
+
+.. important::
+
+    We strongly recommend to use the Yocto's SDK toolchain to create packages as they will be built with the optimization flags suitable to
+    be deployed later to an image generated in a Yocto build.
+
+Deploying an application to a Yocto image
+=========================================
+
+Once you have created and and uploaded the Conan packages to a remote in Artifactory, you can deploy them in a Yocto build.
+
+Prepare your recipes
+--------------------
+
+It is important that in order to accomplish this, that the recipe of the application to be deployed should have a
+`deploy() method <https://docs.conan.io/en/latest/devtools/running_packages.html>`_. There you can specify the files of the application
+needed in the final image as well as any other from its dependencies (like shared libraries or assets):
+
+.. code-block:: python
+
+    def deploy(self):
+        self.copy("*", src="bin", dst="bin")
+        self.copy("*.dll", src="bin", dst="bin")
+        self.copy("*.dylib*", src="lib", dst="bin")
+        self.copy("*.so*", src="lib", dst="bin")
+        self.copy_deps("*.dll", src="bin", dst="bin")
+        self.copy_deps("*.dylib*", src="lib", dst="bin")
+        self.copy_deps("*.so*", src="lib", dst="bin")
+
+Set up the Conan layer
+----------------------
+
+We have created a [meta-conan](LINK) layer that includes all the configuration, the Conan client and a
+generic BitBake recipe. To add the layer you will have to clone it repository and the dependency layers of ``meta-openembedded``:
+
+.. code-block:: bash
+
+    $ cd poky
+    $ git clone LINK
+    $ git clone --branch thud https://github.com/openembedded/meta-openembedded.git
+
+You would also have to activate the layers in the *bblayers.conf* file of your build folder:
+
+.. code-block:: text
+   :caption: *conf/bblayers.conf*
+
+    POKY_BBLAYERS_CONF_VERSION = "2"
+
+    BBPATH = "${TOPDIR}"
+    BBFILES ?= ""
+
+    BBLAYERS ?= " \
+    /home/username/poky/meta \
+    /home/username/poky/meta-poky \
+    /home/username/poky/meta-yocto-bsp \
+    /home/username/poky/meta-openembedded/meta-oe \
+    /home/username/poky/meta-openembedded/meta-python \
+    /home/username/poky/meta-conan \
+    "
+
+After that, a Conan recipe to deploy a Conan package should look as easy as this Bitbake recipe:
```

Comment:
And what about cloning and activating the "conan" layer? if we still don't have a final URL (probably we should have it after the meeting today) put a place holder with XXXXX

Reply:
fixed, will update the link later